### PR TITLE
[Fixed] Missing Package `q` in packages.json & Change dist/main.js

### PR DESCRIPTION
**Pull request by r17x**
I start to learn Vue by your project 

Cheers, 🍺

### CODE REVIEW

**Comment by munirapp** [comment]
I have a question, why you add "q" package in dev dependencies, what it is used for?

**Comment by r17x** [comment]
`q` has depend on this file https://github.com/munirapp/simpel-vue-tps-checker/blob/master/dist/core/DSource.js#L2

File: dist/main.js [hunk 1]
```diff
@@ -2,9 +2,11 @@ import Vue from 'vue';
 import axios from 'axios';
 import DSource from './core/DSource';
 
-window.axios = axios;
-window.vue = Vue;
-window.dataResource = DSource;
+Object.assign({
```
Comment (r17x) [inline dist/main.js:5]: @munirapp  I think 
```
window.axios = ...
window.vue = ...
// Or 
Object.assign(window, {}) 
``` 
is useless. 
maybe we can remove

File: .gitignore [hunk 2]
```diff
@@ -0,0 +1,2 @@
+node_modules
+public
```
Comment (r17x) [inline .gitignore:2]: I add `public` on `.gitignore` because this is has generated by `script` like `yarn dev`. 
this is correct if i removed ? @munirapp

**Comment by munirapp** [comment]
Ah thanks, I was just realized that library used for Pormises in file Dsource.js, I forgot add "q" package because it automatically generated, I will merge it

**Pull request merged**